subql/parachain/kusama/src/helpers/constants.ts

```diff
@@ -0,0 +1,2 @@
+
+export const ALLOW_PARA_IDS = [ 1000 ];
```

Comment:
There need change to correct para id

Reply:
crab parachain's para id is `2105`, please change to 2015 and reset.